packages/protoc-gen-es/README.md

```diff
@@ -1,11 +1,13 @@
 # @bufbuild/protoc-gen-es
 
-The code generator plugin for Protocol Buffers for ECMAScript. Learn more about the project at [github.com/bufbuild/protobuf-es](https://github.com/bufbuild/protobuf-es).
+The code generator plugin for Protocol Buffers for ECMAScript. Learn more about the project at
+[github.com/bufbuild/protobuf-es](https://github.com/bufbuild/protobuf-es).
 
 ## Installation
 
-`protoc-gen-es` generates base types - messages and enumerations - from your Protocol Buffer
-schema. The generated code requires the runtime library [@bufbuild/protobuf](https://www.npmjs.com/package/@bufbuild/protobuf). It is compatible with Protocol Buffer compilers like [buf](https://github.com/bufbuild/buf) and [protoc](https://github.com/protocolbuffers/protobuf/releases).
+`protoc-gen-es` generates base types—messages and enumerations—from your Protocol Buffer
```

Comment:
```suggestion
`protoc-gen-es` generates base types — messages and enumerations — from your Protocol Buffer
```

Not sure if that was intended, but the space around the dashes makes it much easier to read (imo).

Reply:
I tend to agree but I'm all for consistency and leave that up to @rakuista.